Description: Fix zlib related code after repacking
 Fix a few usages of zlib after partial removal through repacking.
Author: Obey Arthur Liu <arthur@milliways.fr>
Author: José Manuel Santamaría Lema <panfaust@gmail.com>
Last-Update: 2012-05-23
--- a/libsrc/Wi/bif_file.c
+++ b/libsrc/Wi/bif_file.c
@@ -4750,7 +4750,11 @@ gz_init_ses (dk_session_t * ses_out)
   level = 6;
 
   err = deflateInit2 (&(s->stream), level,
-      Z_DEFLATED, -MAX_WBITS, DEF_MEM_LEVEL, strategy);
+      Z_DEFLATED, -MAX_WBITS, 9, strategy);
+      // Debian maintainer: was:
+      //Z_DEFLATED, -MAX_WBITS, DEF_MEM_LEVEL, strategy);
+      // DEX_MEM_LEVEL hardcoded at 9, which is the value that
+      // results from upstream shipped zlib build anyway
 
   s->stream.next_out = s->outbuf = (Byte *) dk_alloc (Z_BUFSIZE);
   if (err != Z_OK || s->outbuf == Z_NULL)
--- a/libsrc/Makefile.am
+++ b/libsrc/Makefile.am
@@ -19,7 +19,7 @@
 #  
 #  
 
-SUBDIRS = util zlib odbcsdk Dk Thread langfunc Wi plugin Tidy Xml.new JDBCDriverType4
+SUBDIRS = util odbcsdk Dk Thread langfunc Wi plugin Tidy Xml.new JDBCDriverType4
 
 noinst_HEADERS = Dk.h libutil.h
 
--- a/configure.in
+++ b/configure.in
@@ -2676,7 +2676,6 @@ AC_CONFIG_FILES([
 	libsrc/util/Makefile
 	libsrc/Wi/Makefile
 	libsrc/Xml.new/Makefile
-	libsrc/zlib/Makefile
 	appsrc/Makefile
 	appsrc/ODS-Framework/Makefile
 	appsrc/ODS-Addressbook/Makefile
